Sorts/SwapSort.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @function SwapSort
+ * @description Swap Sort is an algorithm to find the number of swaps required to sort an array.
```

Comment:
Then please rename the function to `minSwapsToSort` or similar

Reply:
updated